test/ui/test_ui_translations.py

```
@@ -0,0 +1,176 @@
+
+    # Give the tabs a moment to render after init completes
+    time.sleep(2)
+
+    # Allow the word 'undefined' inside code/technical blocks but not as a
+    # standalone visible label replacing a translation string.
+    # We check that it does not appear as a trimmed standalone word in headings/tabs.
+    tab_labels = driver.find_elements(By.CSS_SELECTOR, "#tabs-location a, .tab-label, h5")
+    for el in tab_labels:
+        label = el.text.strip()
+        assert label != "undefined", (
+            f"Tab/heading label is literally 'undefined' — plugin strings were not "
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)